libfwupdplugin/fu-security-attrs.c

```diff
@@ -348,6 +348,7 @@ static struct {
     {FWUPD_SECURITY_ATTR_ID_TPM_VERSION_20, FWUPD_SECURITY_ATTR_LEVEL_CRITICAL},
     {FWUPD_SECURITY_ATTR_ID_UEFI_PK, FWUPD_SECURITY_ATTR_LEVEL_CRITICAL},
     {FWUPD_SECURITY_ATTR_ID_UEFI_SECUREBOOT, FWUPD_SECURITY_ATTR_LEVEL_CRITICAL},
+    {FWUPD_SECURITY_ATTR_ID_UEFI_ROLLBACK_PROTECTION, FWUPD_SECURITY_ATTR_LEVEL_SYSTEM_PROTECTION},
```

Comment:
doesn't this need a .xml file so it makes it into the spec? I'm also not sure about the name; it's not UEFI specific.

Reply:
Oh yeah; I'll write up a separate JSON for it.  In terms of the name, any better ideas?  It's for FAR on UEFI firmware, which is why I came up with the name I did.

Reply:
added new JSON file.

Reply:
Renamed the attribute